config/orderly-web.yml

```diff
@@ -57,6 +57,7 @@ orderly:
     tag: master
   env:
     SLACK_URL: VAULT:secret/slack/orderly-webhook:url
+    TEAMS_URL: VAULT:secret/teams/orderly-webhook:url
```

Comment:
Can you please add this to the VIMC vault?

Reply:
Should already be in there - unless I have added it incorrectly?

Reply:
```
vault read secret/teams/orderly-webhook
Key                 Value
---                 -----
refresh_interval    768h
url                 ....
```

Reply:
Sorry, you're right - not sure why that failed for me before 🤔